src/components/TimelineItemList.svelte

```diff
@@ -268,6 +272,9 @@
           style="cursor: move;"
           on:dragend={onDragEnd}
           on:dragstart={e => onDragStart(e.detail, [item])}
+          {tooltip}
```

Comment:
You don't need to pass in the `tooltip` utility function here. You can just import it directly in `ListItem.svelte`